### ISSUE TITLE
Failed to install

### ISSUE THREAD

**Issue by rodymary** [tomjmul/node-red-contrib-fake-wemo#3]
Sorry, I'm not an expert, I got this on a fresh nodered install :

```
npm WARN deprecated node-uuid@1.4.8: Use uuid module instead

> epoll@0.1.22 install /home/pi/.node-red/node_modules/epoll
> node-gyp rebuild

make: Entering directory '/home/pi/.node-red/node_modules/epoll/build'
  CXX(target) Release/obj.target/epoll/src/epoll.o
In file included from ../node_modules/nan/nan.h:192:0,
                 from ../src/epoll.cc:15:
../node_modules/nan/nan_maybe_43_inl.h: In function ‘Nan::Maybe<bool> Nan::ForceSet(v8::Local<v8::Object>, v8::Local<v8::Value>, v8::Local<v8::Value>, v8::PropertyAttribute)’:
../node_modules/nan/nan_maybe_43_inl.h:112:15: error: ‘class v8::Object’ has no member named ‘ForceSet’
   return obj->ForceSet(isolate->GetCurrentContext(), key, value, attribs);
               ^~~~~~~~
In file included from ../src/epoll.cc:15:0:
../node_modules/nan/nan.h: In function ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’:
../node_modules/nan/nan.h:835:60: warning: ‘v8::Local<v8:...
```

**Comment by Blackink90** [comment]
Same problem here, I guess there are some dependencies to install first, but I don't know where to find the list...

**Comment by DENightOne** [comment]
Same problem here, cant install epoll (epoll@0.1.22~install: Failed to exec install script).. Debian 9
it works fine on windows running node-red.... just to test I tryed to install the latest epoll manually and epoll@2.0.8 installs without error.

**Comment by tomjmul** [comment]
I've create a PR for an upstream library that, if merged, will fix this problem.

**Comment by tomjmul** [comment]
Fixed in #4
